include/d/a/d_a_itembase.h

```diff
@@ -0,0 +1,246 @@
+    /* 0x05B */ ITEM_SACRED_BOW = 91,
+    /* 0x05C */ ITEM_BOMB_BAG = 92,
+    /* 0x05D */ ITEM_HEART_CONTAINER = 93,
+    /* 0x05E */ ITEM_HEART_PIECE = 94,
+    /* 0x05F */ ITEM_TRI_COURAGE = 95,
+    /* 0x060 */ ITEM_TRI_POWER = 96,
+    /* 0x061 */ ITEM_TRI_WISDOM = 97,
+    /* 0x062 */ ITEM_SEA_CHART = 98,
+    /* 0x063 */ ITEM_MOGMA_MITTS = 99,
+    /* 0x064 */ ITEM_HEART_MEDAL = 100,
+    /* 0x065 */ ITEM_RUPEE_MEDAL = 101,
+    /* 0x066 */ ITEM_TREASURE_MEDAL = 102,
+    /* 0x067 */ ITEM_POTION_MEDAL = 103,
+    /* 0x068 */ ITEM_CURSED_MEDAL = 104,
+    /* 0x069 */ ITEM_SCATTERSHOT = 105,
+    /* 0x06A */ ITEM_UNK_106 = 106,
+    /* 0x06B */ ITEM_UNK_107 = 107,
+    /* 0x06C */ ITEM_MEDIUM_WALLET = 108,
+    /* 0x06D */ ITEM_BIG_WALLET = 109,
+    /* 0x06E */ ITEM_GIANT_WALLET = 110,
+    /* 0x06F */ ITEM_TYCOON_WALLET = 111,
+    /* 0x070 */ ITEM_ADVENTURE_POUCH = 112,
+    /* 0x071 */ ITEM_POUCH_EXPANSION = 113,
+    /* 0x072 */ ITEM_LIFE_MEDAL = 114,
+    /* 0x073 */ ITEM_UNK_115 = 115,
+    /* 0x074 */ ITEM_WOODEN_SHIELD = 116,
+    /* 0x075 */ ITEM_BANDED_SHIELD = 117,
+    /* 0x076 */ ITEM_BRACED_SHIELD = 118,
+    /* 0x077 */ ITEM_IRON_SHIELD = 119,
+    /* 0x078 */ ITEM_REINFORCED_SHIELD = 120,
+    /* 0x079 */ ITEM_FORTIFIED_SHIELD = 121,
+    /* 0x07A */ ITEM_SACRED_SHIELD = 122,
+    /* 0x07B */ ITEM_DIVINE_SHIELD = 123,
+    /* 0x07C */ ITEM_GODDESS_SHIELD = 124,
+    /* 0x07D */ ITEM_HYLIAN_SHIELD = 125,
+    /* 0x07E */ ITEM_REVITALIZING_POTION = 126,
+    /* 0x07F */ ITEM_REVITALIZING_POTION_PLUS = 127,
+    /* 0x080 */ ITEM_SMALL_SEED_SATCHEL = 128,
+    /* 0x081 */ ITEM_MEDIUM_SEED_SATCHEL = 129,
+    /* 0x082 */ ITEM_LARGE_SEED_SATCHEL = 130,
+    /* 0x083 */ ITEM_SMALL_QUIVER = 131,
+    /* 0x084 */ ITEM_MEDIUM_QUIVER = 132,
+    /* 0x085 */ ITEM_LARGE_QUIVER = 133,
+    /* 0x086 */ ITEM_SMALL_BOMB_BAG = 134,
+    /* 0x087 */ ITEM_MEDIUM_BOMB_BAG = 135,
+    /* 0x088 */ ITEM_LARGE_BOMB_BAG = 136,
+    /* 0x089 */ ITEM_WHIP = 137,
+    /* 0x08A */ ITEM_FIRESHIELD_EARRINGS = 138,
+    /* 0x08B */ ITEM_UNK_139 = 139,
+    /* 0x08C */ ITEM_BIG_BUG_NET = 140,
+    /* 0x08D */ ITEM_FARON_GRASSHOPPER = 141,
+    /* 0x08E */ ITEM_WOODLAND_RHINO_BEETLE = 142,
+    /* 0x08F */ ITEM_DEKU_HORNET_ = 143,
+    /* 0x090 */ ITEM_SKYLOFT_MANTIS = 144,
+    /* 0x091 */ ITEM_VOLCANIC_LADYBUG = 145,
+    /* 0x092 */ ITEM_BLESSED_BUTTERFLY = 146,
+    /* 0x093 */ ITEM_LANAYRU_ANT = 147,
+    /* 0x094 */ ITEM_SAND_CICADA = 148,
+    /* 0x095 */ ITEM_GERUDO_DRAGONFLY = 149,
+    /* 0x096 */ ITEM_ELDIN_ROLLER = 150,
+    /* 0x097 */ ITEM_SKY_STAG_BEETLE = 151,
+    /* 0x098 */ ITEM_STARRY_FIREFLY = 152,
+    /* 0x099 */ ITEM_BOTTLE = 153,
+    /* 0x09A */ ITEM_RUPEE_MEDAL_ALSO = 154,
+    /* 0x09B */ ITEM_HEART_MEDAL_ALSO = 155,
+    /* 0x09C */ ITEM_UNK_156 = 156,
+    /* 0x09D */ ITEM_UNK_157 = 157,
+    /* 0x09E */ ITEM_CAWLINS_LETTER = 158,
+    /* 0x09F */ ITEM_HORNED_COLOSSOL_BEETLE = 159,
+    /* 0x0A0 */ ITEM_BABY_RATTLE = 160,
```

Comment:
```suggestion
    /* 0x09E */ ITEM_CAWLIN_LETTER = 158,
    /* 0x09F */ ITEM_BEEDLE_INSECT_CAGE = 159,
    /* 0x0A0 */ ITEM_RATTLE = 160,
```

This one's just to match the in-game names :p